JSDK-3003 - stop track before restart (#1253)
twilio/twilio-video.js: CHANGELOG.md

Bug Fixes
---------

> - Fixed a bug which caused `restart` method on LocalTrack to fail on some android devices
>   with `Failed to re-acquire MediaStreamTrack`. (JSDK-3003)

Reviewer: ```suggestion
- Fixed a bug where restarting a LocalAudioTrack or LocalVideoTrack failed on some android devices. (JSDK-3003)
```
